src/modules/client/pages/ni/billing/ToBill.vue

```diff
@@ -373,6 +371,16 @@ export default {
     tableRowKey (row) {
       return row.customer._id;
     },
+    async updateDates (date) {
+      try {
+        if (this.billingDatesHasError) return;
```

Comment:
J'ai un soucis que je n'arrive pas a regler: quand les validations renvoient une erreur, l'appel part au back car `billingDatesHasError` n'est pas mis a jour et vaut tjs false.

Ca vient du watcher dans `DateRange` qui ne fonctionne pas. Je ne sais pas trop pourquoi... 
J'ai essayé de supprimer le watcher et de faire `this.$emit('update-error', this.v$.modelValue.$error);` et de recuperer l'evenement dans le composant parent pour mettre a jour "billingDatesHasError" mais il est tjs mis a jour apres billingDates donc l'appel part quand meme et il est bloqué pour la fois d'apres. 

Est-ce que les validaitions du champ ne devraient pas etre gere dans `toBill` ?